python/lsst/fgcmcal/fgcmLoadReferenceCatalog.py

```diff
@@ -324,9 +341,19 @@ def _determine_flux_fields(self, center, filterList):
         self._fluxFields = []
         for filterName in filterList:
             try:
-                fluxField = getRefFluxField(results.refCat.schema, filterName=filterName)
-            except RuntimeError:
-                # This flux field isn't available.  Set to None
+                refFilterName = self.config.refFilterMap[filterName]
+            except KeyError:
+                refFilterName = None
```

Comment:
A bit weird that you set `fluxField` to `None` below, but not in the non-error case.  And if it turns out that's unimportant (it looks like it's set either way in the block below), maybe just change this to
```
refFilterName = self.config.refFilterMap.get(filterName)
```